src/doc/App.scss

```diff
@@ -9,6 +9,7 @@ body {
   position: relative;
   min-height: 100%;
   overflow-y: scroll;
+  font-size: 1.125rem;
```

Comment:
Is that necessary ? does the geops Theme define a good fontSize already?

Reply:
Done